calculate.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once 'database/database.php';
+
+$wyniki = $pdo->query('SELECT * FROM leki')->fetchAll(PDO::FETCH_ASSOC);
+
+foreach ($wyniki as $wynik)
+
+    {
+        $dzisiejsza_data = date("Y-m-d");
+        $data_dodania = $wynik['Data'];
+        $zapotrzebowanie = $wynik['Zapotrzebowanie'];
+
+        $datetime1 = new DateTime($data_dodania);
+        $datetime2 = new DateTime($dzisiejsza_data);
+        $interval = $datetime1->diff($datetime2);
+        $pozostalo = ($interval->format('%R%a days')) * ($zapotrzebowanie);
```

Comment:
Zrób sobie dump co daje Ci `$interval->format('%R%a days')`. Musisz nauczyć się debugować.

Reply:
no obiekt, pozostałe daty też są obiektami....
Co ciekawe na Macu działa normalnie !
pytanie jak to zamienić na inta?

Reply:
$interval->format('%R%a days') daje Ci obiekt?

Reply:
swoją drogą czytanie ze zrozumieniem też by się przydało ;]

Reply:
Jaki konkretnie string?

Reply:
string(8) "+60 days"

Reply:
Czyli mnożysz string przez inta. 

Czy na matematyce dzieci liczą `"dwa muchomor" * 8?` czy `2 * 8` ?

Reply:
już naprawiłem, jest commit